src/ansys/dpf/composites/layup_info/_layup_info.py

```diff
@@ -424,12 +429,14 @@ def get_element_info_provider(
 
     def get_keyopt_property_field(keyopt: int) -> PropertyField:
         keyopt_provider = dpf.Operator("mesh_property_provider")
+        # keyopt_provider = dpf.Operator("property_field_provider_by_name")
```

Comment:
Is this change on purpose? And what about the change on line 439?

Reply:
No, that was an unintentional leftover from testing. Reverted.